src/app/view/components/_common/action-menu-button/action-menu-button.component.html

```diff
@@ -1,5 +1,5 @@
 <span *ngIf="actions.length > 1 && filteredActions.length">
-    <p-menu #menu popup="popup" [model]="filteredActions" class="action-menu__list"></p-menu>
+    <p-menu #menu appendTo="body" popup="popup" [model]="filteredActions" class="action-menu__list"></p-menu>
```

Comment:
@luisrojascr this will fix the issue that the menu was showing in weirds positions.